README.md

```diff
@@ -14,18 +14,95 @@
 
 </div>
 
-Vello is a 2d graphics rendering engine, using [`wgpu`].
+Vello is an experimental 2d graphics rendering engine written in Rust, using [`wgpu`].
 It efficiently draws large 2d scenes with interactive or near-interactive performance.
```

Comment:
I'd probably want some mention of GPU compute quite early on, e.g.

```suggestion
Vello is an experimental 2d graphics rendering engine written in Rust, using GPU compute.
It efficiently draws large 2d scenes with interactive or near-interactive performance.
The reference host code is written in Rust and uses [`wgpu`] for GPU access
```

I don't quite think this flows as well as it needs to, but I think it's useful to show that the GPU code is our main contribution